src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -11357,6 +11394,8 @@ namespace SurfaceGeometry {
 
             ConstrNewSh = TotConstructs + 1;
             SurfaceTmp(SurfNum).ShadedConstruction = ConstrNewSh;
+            WindowShadingControl(WSCPtr).ShadedConstruction = ConstrNewSh;
```

Comment:
The error message you are showing is doing the check you are suggesting. 

I think to accomplish what you are suggesting, the user would simply use the construction rather than the material option in WindowShadingControl.

It probably should have been reported as an error message previously but it wasn't checked as thoroughly before. I didn't add that error message but I think code that I added is applying that check more consistently now. I think it is working as intended. Please explain further what changes you want here.

Reply:
If I understand correctly, previously, each shaded window surface stored it's own shaded and unshaded construction which was set at the time the WindowShadingControl input was processed. So, it the WSC applied a shade material to a list of windows with different base constructions, they all got their own custom shaded construction with that shade layer. Now the shaded construction is being stored on the WSC struct and is getting stepped on by the last unique window construction. Using the shaded construction option in the WSC would require a separate WSC object for every different window type.  All the same checks were there before in `CheckWindowShadingControlFrameDivider`. This breaks files that ran successfully before.

Reply:
Ok. I see what you are saying.